docs/en/observability/synthetics-analyze.asciidoc

```diff
@@ -0,0 +1,358 @@
+[[synthetics-analyze]]
+= Analyze data from synthetic monitors
+
+++++
+<titleabbrev>Analyze monitor data</titleabbrev>
+++++
+
+The Synthetics UI in {kib} both gives you a high-level overview of your service's
+availability and allows you to dig into details to diagnose what caused downtime.
+
+[discrete]
+[[synthetics-analyze-overview]]
+= Overview
+
+The Synthetics *Overview* tab provides you with a high-level view of all the services you are monitoring
+to help you quickly diagnose outages and other connectivity issues within your network.
+
+To access this page, go to *{observability}* -> *Synthetics* and make sure you're on the *Overview* tab.
+
+This overview includes a snapshot of the current status of all monitors, the number of errors that
+occurred over the last 6 hours, and the number of alerts over the last 12 hours.
+All monitors included your deployment will be listed below with information about the location,
```

Comment:
nit I wonder if "deployment" is the right term here?  I wonder if it should be more specific such as _"All Project Monitors or those created in Synthetics, Management..."_